performance/tests/BodyText.test.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
some variables are not defined in `global`
![image](https://user-images.githubusercontent.com/63335823/195343267-00eb0db1-3ca5-4c42-8089-89b678af0b9f.png)